sqlglot/dialects/oracle.py

```diff
@@ -344,3 +356,7 @@ def queryoption_sql(self, expression: exp.QueryOption) -> str:
         def coalesce_sql(self, expression: exp.Coalesce) -> str:
             func_name = "NVL" if expression.args.get("is_nvl") else "COALESCE"
             return rename_func(func_name)(self, expression)
+
+        def into_sql(self, expression: exp.Into) -> str:
+            into = "INTO" if not expression.args.get("bulk_collect") else "BULK COLLECT INTO"
```

Comment:
What about `TEMPORARY` and `UNLOGGED`? It seems like we're generating them in the base method. Are these needed for Oracle?

Reply:
No, `TEMPORARY` and `UNLOGGED` are not used in Oracle's INTO statement.

These are only used in the CTAS case in postgresql.

Reply:
> Perhaps a workaround is to preserve the previous implementation, i.e. set this instead of expressions, if there is only one expression.

@georgesittas Would you please give a suggestion on how to accomplish this?

I tried but was unable to figure it out.

```
expressions = self._parse_expressions()
if len(expressions) == 1:
    ...
```

I think i would need to somehow rewind and then call:

    self._parse_table(schema=True)

Reply:
You can check if `bulk_collect_into` is truthy, in which case you'll set `expressions` to `_parse_csv(self._parse_bitwise)` (*), otherwise just set `this` to `self._parse_table(schema=True)`.

(*): `_parse_expressions` consumes more things and is probably overkill here.

Reply:
Hi @georgesittas ,

For Oracle, INTO and BULK COLLECT INTO can both support multiple values in a csv.

This latest commit seems to solve the issue. If only one expression is found, then I call self._advance(-1), and then revert to the previous implementation.